packages/react-docs/RELEASE-NOTES.md

```diff
@@ -4,6 +4,114 @@ section: get-started
+- [@patternfly/react-tokens@4.9.12](https://www.npmjs.com/package/@patternfly/react-tokens/v/4.9.12)
+- [@patternfly/react-topology@4.6.25](https://www.npmjs.com/package/@patternfly/react-topology/v/4.6.25)
+- [@patternfly/react-virtualized-extension@4.5.105](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/4.5.105)
+
+### Components
+- **Button:** 
+  - Added support for spinner in button ([#4837](https://github.com/patternfly/patternfly-react/pull/4837))
+  - Added support for warning variant ([#4829](https://github.com/patternfly/patternfly-react/pull/4829))
+  - Added support for inline link variation that acts like a link ([#4859](https://github.com/patternfly/patternfly-react/pull/4859))
+- **Chip group:**
+  - Added main div to chip group to fix close button wrapping ([#4873](https://github.com/patternfly/patternfly-react/pull/4873))
+- **Data list:** 
+  - Added draggable feature for basic list ([#4852](https://github.com/patternfly/patternfly-react/pull/4852))
+  - Added support for text modifiers ([#4846](https://github.com/patternfly/patternfly-react/pull/4846))
+  - Added ref to omit list ([#4948](https://github.com/patternfly/patternfly-react/pull/4948))
+  - Updated aria of data list ([#4913](https://github.com/patternfly/patternfly-react/pull/4913))
+- **Date picker:** 
+  Added basic date picker ([#4888](https://github.com/patternfly/patternfly-react/pull/4888))
+- **Dropdown:** 
+  - Fixed indeterminate warning in dropdown toggle checkbox([#4844](https://github.com/patternfly/patternfly-react/pull/4844))
+  - Added more OUIA props into dropdown component ([#4754](https://github.com/patternfly/patternfly-react/pull/4754))
+  - Deprecated the unused openOnEnter prop and removed unused isFocused and isHovered from the dropdown toggle ([#4843](https://github.com/patternfly/patternfly-react/pull/4843))
+- **Label:** 
+  - Added support for React router ([#4934](https://github.com/patternfly/patternfly-react/pull/4934))
+- **Label group:** 
+  - Added label group main div for categories ([#4923](https://github.com/patternfly/patternfly-react/pull/4923))
+- **Modal:** 
+  - Added top align to modal ([#4871](https://github.com/patternfly/patternfly-react/pull/4871))
+- **Nav:**
+  - Added OUIA support into nav item ([#4769](https://github.com/patternfly/patternfly-react/pull/4769))
+- **Notification drawer:** 
+  - Added default variation to notification items ([#4792](https://github.com/patternfly/patternfly-react/pull/4792))
+  - Removed unused import ([#4843](https://github.com/patternfly/patternfly-react/pull/4843))
+- **Page:** 
+  - Replaced hasNoPadding property in the documentation example ([#4848](https://github.com/patternfly/patternfly-react/pull/4848))
+  - Fixed handleMainClick function signature ([#4843](https://github.com/patternfly/patternfly-react/pull/4843))
+- **Progress:** 
+  - Added warning variant ([#4834](https://github.com/patternfly/patternfly-react/pull/4834))
+- **Progress:** 
```

Comment:
Can remove duplicate "Progress" heading and move this bullet up with the other Progress item

```suggestion

```